### PR TITLE
fixed the bug on no-rules in config

### PR DESCRIPTION
**Pull request by XinyuLuan**

### CODE REVIEW

File: moesifdjango/moesif_gov.py [hunk 1]
```diff
@@ -15,32 +15,37 @@ def __init__(self):
...
-    def fetch_entity_rules_from_app_config(cls, config):
+    def fetch_entity_rules_from_app_config(cls, config, debug):
         """
         get fetch entity rules from config
         :param config:
-        :return:
+        :return: None, if there is no rules in config (no dynamic variable set with gov rules with the app)
+                Otherwise, return mapping with keys user_rules and company_rules
         """
         entity_rules = {}
```
Comment (praves77) [inline moesifdjango/moesif_gov.py:25]: Set the `entity_rules` as 
`entity_rules = {"user_rules": {}, "company_rules" : {} }`

File: moesifdjango/moesif_gov.py [hunk 2]
```diff
@@ -15,32 +15,37 @@ def __init__(self):
...
-                print(f"[moesif] {raw_body}'s attribute [company_rules] is unknown.")
+                entity_rules['company_rules'] = company_rules
+            else:
+                entity_rules['company_rules'] = {}
+                if debug:
+                    print("[moesif] No attribute [company_rules] is found in ", raw_body)
 
         except KeyError:
-            print(f"[moesif] config attribute ['raw_body'] is unknown.")
+            print("[moesif] config attribute ['raw_body'] is not found.")
```
Comment (bakennedy) [inline moesifdjango/moesif_gov.py:48]: I think the behavior will be clearer and more idiomatic python if the above logic is replaced with:
```
entity_rules = {
    'user_rules': raw_body.get('user_rules', {}),
    'company_rules': raw_body.get('company_rules', {}),
}
print(f"[moesif] config got {len(entity_rules['user_rules'])} cohort users and {len(entity_rules['company_rules'])} cohort companies")
return entity_rules
```

File: moesifdjango/moesif_gov.py [hunk 3]
```diff
@@ -15,32 +15,37 @@ def __init__(self):
         pass
 
     @classmethod
-    def fetch_entity_rules_from_app_config(cls, config):
+    def fetch_entity_rules_from_app_config(cls, config, debug):
         """
         get fetch entity rules from config
         :param config:
-        :return:
+        :return: None, if there is no rules in config (no dynamic variable set with gov rules with the app)
```
[the remark this answers is not in the document]
Comment (XinyuLuan) [reply moesifdjango/moesif_gov.py:22]: I checked it before it we used the `entity_rules` [here](https://github.com/Moesif/moesifdjango/blob/871d5608159a1c1f55b868699ae005a12f970d00/moesifdjango/moesif_gov.py#L551), so if no rules in config, then we will not do anything on entity rules.

**Review by bakennedy — APPROVED** [review]
Just a nit on simplifying some logic and reducing log verbosity on expected, non-error conditions in the response

File: moesifdjango/moesif_gov.py (hunk 1 above)
Comment (XinyuLuan) [reply moesifdjango/moesif_gov.py:25]: Changed

File: moesifdjango/moesif_gov.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (XinyuLuan) [reply moesifdjango/moesif_gov.py:22]: Changed

File: moesifdjango/moesif_gov.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (XinyuLuan) [reply moesifdjango/moesif_gov.py:48]: Changed to `Exception`

Comment (XinyuLuan) [reply moesifdjango/moesif_gov.py:48]: Changed, Thanks a lot

**Review by keyur9 — APPROVED** [review]
lgtm

**Review by praves77 — APPROVED** [review]

**Pull request merged**